auxprogs/homGeneMapping/src/Makefile

```diff
@@ -12,7 +12,7 @@ CC      = g++
 # Notes: - "-Wno-sign-compare" eliminates a high number of warnings (see footnote below). Please adopt
 #          a strict signed-only usage strategy to avoid mistakes since we are not warned about this.
 #        - The order of object files in $(OBJS) IS IMPORTANT (see lldouble.hh)
-CFLAGS := -Wall -Wno-sign-compare -ansi -pedantic -std=c++0x -pthread -O2 ${CFLAGS} # -DDEBUG -g -ggdb -pg
+CPPFLAGS := -Wall -Wno-sign-compare -ansi -pedantic -std=c++0x -pthread ${CPPCLAGS}
```

Comment:
- in this file macro CC is defined as g++ -> better use CXX ?
- CPPFLAGS should renamed to CXXFLAGS as it contains flags for the C++ compiler
- later CFLAGS should no longer used but CXXFLAGS (if not already there)